packages/server/src/plugin/usageReporting/stats.ts

```diff
@@ -157,10 +168,15 @@ class StatsByContext {
     }
   }
 
-  addTrace(trace: Trace, sizeEstimator: SizeEstimator) {
+  addTrace(
+    trace: Trace,
+    sizeEstimator: SizeEstimator,
+    nonFtv1ErrorPaths: { subgraph: string; path: GraphQLError['path'] }[],
```

Comment:
perhaps `GraphQLRequestMetrics['nonFtv1ErrorPaths']` if you don't want to give it a name (and in a couple other places)

Reply:
52f8b6494d7009b1dfadf3b4f830854f44f50585